lightning/src/chain/channelmonitor.rs

```diff
@@ -687,7 +708,9 @@
 	// Used to track onchain events, i.e transactions parts of channels confirmed on chain, on which
 	// we have to take actions once they reach enough confs. Key is a block height timer, i.e we enforce
 	// actions when we receive a block with given height. Actions depend on OnchainEvent type.
-	onchain_events_waiting_threshold_conf: HashMap<u32, Vec<OnchainEvent>>,
+	//
+	// TODO: Rewrite docs
+	onchain_events_waiting_threshold_conf: Vec<OnchainEventEntry>,
```

Comment:
nit, but it should be either events_waiting_for_threshold_conf, or events_awaiting_threshold_conf